src/engines/html5/capabilities/html5-autoplay.js

```diff
@@ -1,35 +1,111 @@
 // @flow
 import * as Utils from '../../../utils/util'
 import * as EncodingSources from '../../../assets/encoding-sources.json'
+import {HTML5_EVENTS as Html5Events} from '../../../event/events'
 
-export default class Html5AutoPlayCapability implements ICapability {
+/**
+ * The time (miliseconds) we wait that the playing event will be fired
+ * @const
+ */
+const playTimeoutBuffer = 1500;
+
+export default class Html5AutoPlayCapabilities implements ICapability {
   static _vid: HTMLVideoElement = Utils.Dom.createElement('video');
-  static _promise: Promise<*>;
+  static _mutedVid: HTMLVideoElement = Utils.Dom.createElement('video');
+  static _autoPlayPromise: Promise<*>;
+  static _mutedAutoPlayPromise: Promise<*>;
 
   /***
-   * Runs the test for autoplay capability.
+   * Runs tests for autoplay & mutedAutoplay capability.
    * @public
    * @static
    * @returns {void}
    */
   static runCapability(): void {
+    Html5AutoPlayCapabilities._runAutoplayCapabilty();
+    Html5AutoPlayCapabilities._runMutedAutoplayCapability();
+  }
+
+  /***
+   * Runs test for autoplay capability.
+   * @static
+   * @returns {void}
+   */
+  static _runAutoplayCapabilty(): void {
+    try {
+      Html5AutoPlayCapabilities._vid.src = EncodingSources.Base64Mp4Source;
+      Html5AutoPlayCapabilities._autoPlayPromise = Html5AutoPlayCapabilities._vid.play() || Html5AutoPlayCapabilities._oldBrowsersCapability(Html5AutoPlayCapabilities._vid);
```

Comment:
@odedhutzler need to verify that this won't cause audible playback

Reply:
@OrenMe  how can it? the browser won't allow it (?)

Reply:
if browser allows it this test will cause sound

Reply:
@OrenMe so we need another method to check this capability, that's the way it checks autoplay in the current version of the player.